corehq/form_processor/submission_post.py

```diff
@@ -429,6 +430,8 @@ def do_post_save_actions(case_db, xforms, case_stock_result):
         try:
             case_stock_result.stock_result.finalize()
 
+            SubmissionPost.send_to_elasticsearch(instance)
```

Comment:
you could pass the cases in directly instead of having to re-fetch them (see line below: `case_stock_result.case_models`)
```suggestion
            SubmissionPost.send_to_elasticsearch(case_stock_result.case_models)
```

Reply:
8d5e3760f01 - I saw "stock" earlier and assumed it was just something for commtrack